src/components/client/common/header/index.jsx

```diff
@@ -1,58 +1,63 @@
-import React from 'react'
 import './header.css'
 import logo from '../../../../assets/logo/FCSCMedia.png'
```

Comment:
try to use webp images

Reply:
![image](https://user-images.githubusercontent.com/85289620/226543389-422fc858-905d-4abb-87ff-7df3a7e94b61.png)